tox.ini

```diff
@@ -1,32 +1,32 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,py36,pypy,pypy3,coverage,docs
+    py27,py34,py35,py36,pypy,pypy3
```

Comment:
- I'm not willing to have remote CI failures be the only way we find out about coverage gaps.  Seems like `detox` needs to get some "run at the end" help, maybe.
- `tox -e docs` is not just about doctests:  it is about ensuring that the actual Sphinx docs build correctly.